### ISSUE TITLE
0.30.0 - active gateway shouldn't be blocked by lack of entry in enforced known list

### ISSUE THREAD

**Issue by zxdavb** [zxdavb/ramses_cc#73]
```
2023-11-11 21:12:57.188 WARNING (MainThread) [ramses_tx.transport] Active gateway (by signature) not in known_list: 18:006402, configure the known_list/block_list as required
2023-11-11 21:12:57.190 WARNING (MainThread) [ramses_rf.dispatcher]  I --- 18:006402 63:262142 --:------ 7FFF 015 0010018BC03B0C0976302E33302E30 < LookupError(Can't create 18:006402: it is not an allowed device_id (if required, add it to the known_list))
```

The packet above is being ignored, courtesy of the device id filters.

Thus, detection of the HGI device address by signature fails, because the packed is never processed.

Solution, these signature packets should not be filtered.

**Comment by zxdavb** [comment]
This is the relevant code:
```python
if dev_id in self._exclude:
    _LOGGER.error(f"{msg} is in {SZ_BLOCK_LIST}: {dev_id}{TIP}")
elif dev_id in self._include:
    _LOGGER.info(f"{msg}: {dev_id}")
else:
    _LOGGER.warning(f"{msg} not in {SZ_KNOWN_LIST}: {dev_id}{TIP}")
self._extra[SZ_ACTIVE_HGI] = dev_id
```
.. the consequence of `self._extra[SZ_ACTIVE_HGI]` not being None is:
```python
if dev_id == src_id and payload == self._extra[SZ_SIGNATURE]:
    self._set_active_hgi(dev_id)
elif dev_id == self._extra[SZ_KNOWN_HGI]:
    self._set_active_hgi(dev_id)
```
That is, this is only an issue if auto-detection of the HGI address (by signature) fails.

**Comment by zxdavb** [comment]
I believe has been fixed by commit 73d41442 in the **ramses_rf** repo.
